pkg/cloudprovider/providers/aws/aws_loadbalancer.go

```diff
@@ -28,7 +28,9 @@ import (
 	"k8s.io/kubernetes/pkg/util/sets"
 )
 
-func (s *AWSCloud) ensureLoadBalancer(namespacedName types.NamespacedName, loadBalancerName string, listeners []*elb.Listener, subnetIDs []string, securityGroupIDs []string, internalELB bool) (*elb.LoadBalancerDescription, error) {
+const ProxyProtocolPolicyName = "k8s-proxyprotocol"
```

Comment:
How about we call this "k8s-proxyprotocol-enabled", as the value is always true and when false we just don't attach it.